src/actions/ui.js

```diff
@@ -1,18 +1,27 @@
 // @flow
-import { getSource, getActiveSearchState } from "../selectors";
+import { getSource, getActiveSearch } from "../selectors";
 import type { ThunkArgs } from "./types";
 import type { ActiveSearchType, SymbolSearchType } from "../reducers/ui";
+import { clearSourceSearchQuery } from "./source-search";
 
 export function closeActiveSearch() {
-  return {
-    type: "TOGGLE_ACTIVE_SEARCH",
-    value: null
+  return ({ getState, dispatch }: ThunkArgs) => {
+    const activeSearch = getActiveSearch(getState());
+
+    if (activeSearch == "source") {
```

Comment:
I am not sure I understand this.
Why do we check for `source` ? A comment, or using a constant for this `"source"` value would help understanding